tests/TestHelpers/OcisHelper.php

```diff
@@ -47,18 +47,11 @@ public static function isTestingOnReva():bool {
 		return (\getenv("TEST_REVA") === "true");
 	}
 
-	/**
-	 * @return bool
-	 */
-	public static function isTestingOnOcisOrReva():bool {
-		return (self::isTestingOnOcis() || self::isTestingOnReva());
-	}
-
 	/**
 	 * @return bool
 	 */
 	public static function isTestingOnOc10():bool {
-		return (!self::isTestingOnOcisOrReva());
+		return false;
 	}
```

Comment:
is this function used anywhereelse in `ocis`?

Reply:
@grgprarup This PR is just to remove `isTestingOnOcisOrReva` method and its use. There are lots of things to remove and will do in other PRs.